Sources/ProjectDescription/Cloud.swift

```diff
@@ -24,6 +24,7 @@ public struct Cloud: Codable, Equatable, Sendable {
     ///   - url: Base URL to the Cloud server.
     ///   - options: Cloud options.
     /// - Returns: A Cloud instance.
+    @available(*, deprecated, message: "Use the `fullHandle` and `url` properties directly in the `Config`")
```

Comment:
Should we also show a runtime warning when we parse the configuration and detect that they are using the old API. That way they'll get the warning when running CLI commands.

Reply:
I'd keep `ProjectDescription` deprecations in `ProjectDescription` only.